Recipes.Test/JsonConfiguration/DefaultJsonSerializerTest.cs

```diff
@@ -0,0 +1,137 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DefaultJsonSerializerTest.cs">
+//     Copyright (c) 2016. All rights reserved. Licensed under the MIT license. See LICENSE file in
+//     the project root for full license information.
+// </copyright>
+// <auto-generated>
+// Sourced from NuGet package. Will be overwritten with package update except in Spritely.Recipes source.
+// </auto-generated>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Spritely.Recipes.Test
+{
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DefaultJsonSerializerTest
+    {
+        [Test]
+        public void SerializeObject_without_type_serializes_to_json_using_DefaultSerializerSettings()
+        {
+            var dog = new Dog(5, "spud", FurColor.Brindle);
+
+            var json = DefaultJsonSerializer.SerializeObject(dog);
+
+            Assert.That(json, Is.EqualTo("{\r\n  \"name\": \"spud\",\r\n  \"furColor\": \"brindle\",\r\n  \"dogTag\": null,\r\n  \"age\": 5\r\n}"));
+        }
+
+        [Test]
+        public void SerializeObject_with_type_serializes_to_json_using_DefaultSerializerSettings()
+        {
+            var dog = new Dog(5, "spud", FurColor.Brindle);
+
+            var json = DefaultJsonSerializer.SerializeObject(dog, typeof(Animal));
+
+            Assert.That(json, Is.EqualTo("{\r\n  \"name\": \"spud\",\r\n  \"furColor\": \"brindle\",\r\n  \"dogTag\": null,\r\n  \"age\": 5\r\n}"));
+        }
+
+        [Test]
+        public void DeserializeObject_generic_deserializes_json_using_DefaultSerializerSettings()
+        {
+            var dogJson = "{\"name\":\"Barney\",\"furColor\":\"brindle\",\"age\":10}";
```

Comment:
Yup - totally see your point, I'm just saying that it doesn't matter.  It was easier to encode this as Compact because I didn't have to do the newlines.